docs/destinations/warehouses-and-lakes/index.md

```diff
@@ -4,7 +4,40 @@ sidebar_position: 2
 ---
 
 ```mdx-code-block
-import DocCardList from '@theme/DocCardList';
-
-<DocCardList />
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 ```
+
+To store your enriched Snowplow data, you will need to determine which loader to set-up to ensure your enriched data reaches your data warehouse or lake. 
+
+### Selecting a loader for your data warehouse
+#### Selecting a loader by destination and cloud provider
```

Comment:
I believe this header might now be redundant

Reply:
Agreed. Thanks for catching that.